src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -61,9 +58,10 @@ class SaveInProgressErrorPage extends React.Component {
   };
 
   render() {
-    const { loadedStatus } = this.props;
-    const { forbidden, noAuth, notFound } =
-      this.props.route.formConfig.savedFormMessages || {};
+    const { loadedStatus, formConfig } = this.props;
+    const { forbidden, noAuth, notFound } = formConfig.savedFormMessages || {};
+    const continueAppMessage =
+      formConfig.continueAppMessage || 'Continue your application';
```

Comment:
```suggestion
      formConfig.continueAppMessage || 'Continue Your Application';
```
We should keep the same casing as the message we're replacing.

Reply:
Fixed